data/primer3/presets/Default.txt

```diff
@@ -43,7 +43,7 @@ SEQUENCE_FORCE_LEFT_START=-1000000
 SEQUENCE_FORCE_RIGHT_END=-1000000
 SEQUENCE_FORCE_RIGHT_START=-1000000
 SEQUENCE_START_CODON_POSITION=-2000000
-PRIMER_ANNEALING_TEMP=0
+PRIMER_ANNEALING_TEMP=-10
```

Comment:
Why do we have this as the default value? What does -10 means?

Reply:
This is the default value used by the original Primer3 tool (see [here](https://github.com/primer3-org/primer3/blob/7f9f17d6012f404e83bbf0f931a4d06eb4af465b/src/libprimer3.cc#L702)). We had 0 previusly, because `QDoubleSpinBox`, responsible for this value, had 0 as a minimal limit. When I created this ".txt" file I just saved parameters from the Primer3 dialog and didn't notice this mistake. In this commit I fixed limits for some spinboxed (this one was included) and changed the default value as it set in Primer3.

If you are asking about the biological sence of this variable, I'm not sure that I'm the one who may give you a good answer, I understand it very approximatelly.